web3-utils/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14-alpine
+
+WORKDIR /app
+
+RUN apk add git
+
+COPY package.json .
+
+RUN yarn install
+
+COPY . .
+
+# ENTRYPOINT yarn deploy
+# CMD node index.js
+CMD ./node_modules/.bin/ts-node index.ts && \
+    ./node_modules/.bin/truffle compile && \
```

Comment:
Why do we need truffle? If he already have web3, can't we simply compile the contracts with `solc`?

Reply:
Originally this PR had ganache + truffle.  I asked @NoCtrlZ if he could replace ganache with geth, because I thought it would be more useful in the long run.
That's why I think truffle is still there.  I agree that using truffle just to compile is overkill and it could be replaced by `solc`.

Reply:
I replaced with solc!